crates/router/src/core/payments/operations/payment_response.rs

```diff
@@ -1377,6 +1454,87 @@ async fn payment_response_update_tracker<F: Clone, T: types::Capturable>(
                                     .payment_intent
                                     .fingerprint_id
                                     .clone_from(&payment_data.payment_attempt.fingerprint_id);
+
+                                if let Some(payment_method) =
```

Comment:
can we just clone the required field and not the whole object

Reply:
No we cannot do that